x-pack/plugins/fleet/server/errors/handlers.ts

```diff
@@ -51,6 +52,10 @@ export const isLegacyESClientError = (error: any): error is LegacyESClientError
   return error instanceof LegacyESErrors._Abstract;
 };
 
+export function isESClientError(error: any): error is ResponseError {
+  return Boolean(error.body && error.statusCode && error.headers);
```

Comment:
__note:__ Core team is going to verify if we can just use `error instanceof ResponseError`. I'll update this helper if it's the case.

Reply:
Confirmed and changed to `instanceof`!